spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -47,6 +47,42 @@
         click_idv_continue
       end
 
+      context 'when the user times out waiting for results' do
```

Comment:
this test is great ... can we also test Try Again button? ... if possible, in the same spec would be 👍🏿

Reply:
Done in 5b7562ed57